tests/test_skill_prompt.py

```
@@ -3,12 +3,14 @@
 import pytest
 
 from holmes.core.prompt import generate_user_prompt
-from holmes.utils.global_instructions import generate_runbooks_args
+from holmes.utils.global_instructions import generate_skills_args
 
 
-class DummyRunbookCatalog:
+class DummySkillCatalog:
+    skills = [True]  # non-empty so getattr check passes
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)